packages/loot-core/src/server/budget/goaltemplates.ts

```diff
@@ -104,8 +156,7 @@ async function processTemplate(month, force) {
   }
 }
 
-const TEMPLATE_PREFIX = '#template ';
-
+const TEMPLATE_PREFIX = '#template';
```

Comment:
I had the space here so that people couldn’t write e.g. `#templatefoo bar` — how are you addressing that now?

Reply:
The updated parser looks for either a priority flag or whitespace after the template tag.  Anything else fails the parsing and is ignored like other bad templates.

Reply:
Thank you for clarifying!